### PR TITLE
ETC Course: 30. What Are File Storage Coins?

### PR DESCRIPTION
**Pull request by DonaldMcIntyre**
This is class 30 of the ETC course that explains file storage coins. It is the last class about the categories of blockchains and coins.

"Ethereum Classic Course: 30. What Are File Storage Coins?"

EN.

CN coming soon.

This one will be published on July 13th so we have a 13 day lead.

### CODE REVIEW

File: content/blog/2023-07-13-ethereum-classic-course-30-what-are-file-storage-coins/index.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+## Examples of File Storage Coins: Filecoin
+
+![Filecoin.](./3.png)
+
+Filecoin is a blockchain that uses the IPFS protocol for file storage and has a ledger with the “FIL” coin that is used to pay for providers. It has three main types of users: Content users and uploaders, storage providers, and retrieval providers.
+
+Storage providers are IPFS protocol nodes, but they actually get paid in FIL by users to store their files. IPFS and Filecoin are two different networks that use the IPFS protocol (IPFS just happens to have the same name because it was started by the same founder who started Filecoin).
+
+Storage providers must inform with a certain frequency to the peer-to-peer network of the integrity of the data they store as a proof of storage. Content users are the ones who use the network to store their data. Retrieval providers are like brokers that get paid to be in touch with users and retrieve from storage providers the most popular content units to deliver them faster to users.
```
Comment (TheCrowbill) [inline content/blog/2023-07-13-ethereum-classic-course-30-what-are-file-storage-coins/index.md:63]: Line 63:  Revise for clarity
```suggestion
As proof of storage, storage providers must inform the peer-to-peer network of the integrity of the data they store on a regular basis. Users are the ones who store their data on the network. Retrieval providers are like brokers who get paid to communicate with users and retrieve popular content from storage providers for quicker delivery. 
```

File: content/blog/2023-07-13-ethereum-classic-course-30-what-are-file-storage-coins/index.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+In essence Filecoin is like a marketplace where data storage around the world is put for sale and the protocol acts as the pricing and payment system where users compensate storage and retrieval providers for their content to be stored and delivered efficiently.
```
Comment (TheCrowbill) [inline content/blog/2023-07-13-ethereum-classic-course-30-what-are-file-storage-coins/index.md:65]: Line 65: Revise for clarity
```suggestion
In essence, Filecoin is a marketplace where data storage around the world is put up for sale and the protocol acts as the pricing and payment system where users compensate storage and retrieval providers for their content to be stored and delivered efficiently. 
```

File: content/blog/2023-07-13-ethereum-classic-course-30-what-are-file-storage-coins/index.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+The current market capitalization of FIL at the time of this writing is $1.70 billion.
+
+## Examples of File Storage Coins: Bittorrent
+
+![BitTorrent.](./4.png)
+
+BitTorrent is a pioneering file sharing protocol and network that started in the early 2000s. It was acquired by the TRON Foundation in 2018 to integrate the “BTT” token to its ecosystem with the goal of creating a decentralized file sharing system for web3 and streaming services.
+
+It is similar to Filecoin in that it is a content addressed network, and content units are broken up in chunks and stored in many peers in the system. BTT is the token that is used in this case.
```
Comment (TheCrowbill) [inline content/blog/2023-07-13-ethereum-classic-course-30-what-are-file-storage-coins/index.md:77]: Line 77: Revise for  clarity
```suggestion
It is similar to Filecoin in that it is a content-addressed network where content units are broken up into chunks and stored by many peers in the system. BTT is the token that is used in this case. 
```

File: content/blog/2023-07-13-ethereum-classic-course-30-what-are-file-storage-coins/index.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+Link: 
+
+https://messari.io/screener/file-storage-A75CCE80
+
+Filecoin and BitTorrent are the largest and most well known coins in the sector.
+
+## Uses for Web3
+
+As we will see in the next 8 classes of the Ethereum Classic course, the blockchain industry will be divided in layers and many components will exist in each layer. These layers will be organized by security vs scalability degrees where the most secure but less scalable proof of work coins will constitute the base layer, and the rest of the more scalable but less secure systems, including proof of stake, will exist on top of the security layer.
```
Comment (TheCrowbill) [inline content/blog/2023-07-13-ethereum-classic-course-30-what-are-file-storage-coins/index.md:105]: Line 106 Correct usage;
```suggestion
As we will see in the next 8 classes of the Ethereum Classic course, the blockchain industry will be divided into layers and many components will exist in each layer. These layers will be organized by security vs scalability degrees where the most secure but less scalable proof of work coins will constitute the base layer, and the rest of the more scalable but less secure systems, including proof of stake, will exist on top of the security layer.
```

File: content/blog/2023-07-13-ethereum-classic-course-30-what-are-file-storage-coins/index.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+The combination of all these components, plus the many more types of blockchains that exist, will create for the end user the full experience of the web as we know it today, but much more secure. 
+
+Web3 is a term that comes after the static web1 of the 90s, and the social media centered web2 of the 2000s.
+
+## Risks of File Storage Coins
+
+However, even though file storage networks and coins use decentralized ledgers and peer-to-peer protocols to manage large amounts of data, this does not mean that they are completely decentralized.
+
+The file storage protocols side of these networks are just peer-to-peer file systems where the files are not actually stored in all the nodes, but just those who want to store them as they expect to get paid for serving them.
```
Comment (TheCrowbill) [inline content/blog/2023-07-13-ethereum-classic-course-30-what-are-file-storage-coins/index.md:123]: Line 131: Agreement
```suggestion
The file storage protocol side of these networks is just peer-to-peer file systems where the files are not actually stored in all the nodes, but just those who want to store them as they expect to get paid for serving them. 
```

File: content/blog/2023-07-13-ethereum-classic-course-30-what-are-file-storage-coins/index.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+Web3 is a term that comes after the static web1 of the 90s, and the social media centered web2 of the 2000s.
+
+## Risks of File Storage Coins
+
+However, even though file storage networks and coins use decentralized ledgers and peer-to-peer protocols to manage large amounts of data, this does not mean that they are completely decentralized.
+
+The file storage protocols side of these networks are just peer-to-peer file systems where the files are not actually stored in all the nodes, but just those who want to store them as they expect to get paid for serving them. 
+
+On the other hand, the decentralized ledgers that hold the accounts and balances of the file storage coins and conduct the payments for file storage providers are all proof of stake or proof of authority systems. These are much less secure that proof of work networks.
```
Comment (TheCrowbill) [inline content/blog/2023-07-13-ethereum-classic-course-30-what-are-file-storage-coins/index.md:125]: Line 125: Spelling typo
```suggestion
On the other hand, the decentralized ledgers that hold the accounts and balances of the file storage coins and conduct the payments for file storage providers are all proof of stake or proof of authority systems. These are much less secure than proof of work networks.
```

**Review by TheCrowbill — CHANGES_REQUESTED** [review]

**Review by brolal — APPROVED** [review]

**Review by TheCrowbill — APPROVED** [review]

**Pull request merged**